src/wallet/init.cpp

```diff
@@ -168,6 +171,10 @@ bool WalletInit::ParameterInteraction() const
         return InitError(strprintf(_("%s can't be lower than %s"), "-coinjoindenomshardcap", "-coinjoindenomsgoal"));
     }
 
+    if (CMnemonic::Generate(gArgs.GetArg("-mnemonicbits", CHDChain::DEFAULT_MNEMONIC_BITS)) == SecureString()) {
```

Comment:
note: `CMnemonic::Generate` validates amount of bits:
```
if (strength % 32 || strength < 128 || strength > 256) ...
```

So, it's impossible to specify accidentally 16 bits or somewhat like that. 

--------
Btw, I don't see functional/unit tests for that. Should we have?

Reply:
may be we can also expand help message such as:
"Invalid %s" -> "Invalid number '%s': should be in [128...256] and multiple of 32"?